src/frontends/pytorch/src/op/cat.cpp

```diff
@@ -94,6 +95,24 @@ OutputVector translate_quantized_cat(const NodeContext& context) {
                      list_elems.front())};
 };
 
+OutputVector translate_stack(const NodeContext& context) {
```

Comment:
This is fx speccific translator, better call it *_fx

Reply:
Done